utils/docker/vcluster/README.md

```diff
@@ -24,18 +24,19 @@ At this time only emulated hardware storage are supported by this Docker platfor
 
 To build and deploy the Docker images, `docker` and optionally `docker-compose` shall be available.
 The docker host should have access to the [Docker Hub](https://hub.docker.com/) and
-[CentOS](https://www.centos.org/) official repositories.  Finally,
+[Rocky Linux](https://rockylinux.org/fr/) official repositories.  Finally,
 [hugepages](https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt) linux kernel feature shall
 be enabled on the docker host.
 
-The platform was tested and validated with the [CentOS8](https://hub.docker.com/_/centos) official
-docker image.  However other RHEL-like distributions such as
-[rocky8.4](https://hub.docker.com/r/rockylinux/rockylinux) should be supported.
+The platform was tested and validated with the
+[rockylinux/rockylinux:8.4](https://hub.docker.com/r/rockylinux/rockylinux) and
+[centos:centos8](https://hub.docker.com/_/centos) official docker images.  However other RHEL-like
+distributions should be supported.
 
 !!! warning
     Some distributions are not yet well supported such as
-    [rocky8.5](https://hub.docker.com/r/rockylinux/rockylinux): issue with the management of
-    hugepages with the [spdk](https://spdk.io/) library.
+    [rockylinux/rockylinux:8.5](https://hub.docker.com/r/rockylinux/rockylinux): issue with the
```

Comment:
Almost all of this PR refers to 8.4, this line however says 8.5

Reply:
As indicated in my previous comment, this sentence sadly indicate that we are not yet supporting rocky8.5.

Reply:
Ah, so the issue here is not with Rocky it's with the 8.5 version of rocky.  I'd misunderstood this comment.

Reply:
yeah, baby steps, but we will get there.